uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-request-modal.scss

```diff
@@ -133,6 +133,13 @@ $row-margin-bottom: 15px;
   margin-top: 7px;
 }
 
+.chr_leave-request-modal__expiry-clear {
```

Comment:
what's this? You are not mentioning this in the PR description

Reply:
It was part of the reverted code. It got reverted to its previous state.